multi_press/automation/jsr223/python/community/multi_press/multi_press.py

```diff
@@ -0,0 +1,178 @@
+"""
+Copyright September 23, 2020 Sascha Volkenandt
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+import traceback
+import core
```

Comment:
I don't think this line does anything.

Reply:
core is needed to set core.MultiPressProfileFactory in line 152ff. Although I don't see the need to place that object in the core package (again, taken from 200_JythonTransform.py). I'll try to place it inside the script itself...